05_syntax_lab/01.py

```diff
@@ -1,5 +1,19 @@
-"""
-Write a program that reads 10 numbers from
-the user and prints the largest one
-"""
-
+"""
+
+write program that gets from user 10 numbers and prints the biggest
+
+
+"""
+
+
+max =0
+
+for i in range (1,11):
+
+	num = int(raw_input())
+	if (num>max):
+		max=num

```

Comment:
Notice how github colours "max" in the same color as print (and differently from "num"). The reason is python has a function named max, so when you use that as a variable name it can be a bit confusing.